### DISCUSSION TITLE
Pages not showing

### DISCUSSION THREAD

**Discussion by ghost** [thuliteio/doks#1004] [Q&A]
Hey,

I am new to Hugo as I decided to transition from Jekyll recently. I am currently trying to port my documentation as it is too big and I head Hugo’s performance is much better.

I decided to go with Doks theme. I am experiencing issues though and I am unable to see what is the problem.

What I have done:

- I have copied the Child theme according to documentation.
- `npm run create -- --kind docs [section]` with api and articles as name for sections
- I have tried adding some pages under articles => general => introduction as an example
- Also, I edited menu config

The problem is pages are not showing in my articles as well. I am trying to recreate what you can see in Docs tab. I am nesting the pages in folders as there will be images in these articles.

Example of desired result:
![image](https://user-images.githubusercontent.com/49745287/218152428-413b5388-827d-4ae8-aad9-0038742b10eb.png)

Repository
[Natromitus/tsl2-doc-another-try (github.com)](https://github.com/Natromitus/tsl2-doc-another-try)

**Comment by h-enk** [comment]
![Snag_1a20d00f](https://user-images.githubusercontent.com/3902872/218249522-1864aae7-5b1e-4a95-964a-67a9dcbb8121.png)


To get the "Other articles" page:

1. Copy the `student-intro` folder and paste it as `other-articles`

2. Update the `title`, `identifier`, and `weight` of `other-articles/index.md`

    ```md
    ---
    title: "Other articles"
    description: ""
    lead: ""
    date: 2023-02-10T14:41:39+01:00
    lastmod: 2023-02-10T14:41:39+01:00
    draft: false
    images: []
    type: docs
    menu:
      articles:
        parent: "general"
        identifier: "other-articles"
    weight: 210
    toc: true
    ---
    ```

To get the "Setup guides" folder:

1. Copy the `general` folder and paste it as `setup-guides`

2. Update the `title` of  `setup-guides/_index.md`

    ```md
    ---
    title: "Setup Guides"
    description: ""
    lead: ""
    date: 2022-01-25T14:41:21+01:00
    lastmod: 2022-01-25T14:41:21+01:00
    draft: false
    images: []
    type: docs
    ---
    ```

3. Update the `title` , `parent`, `identifier`, and `weight` of  `setup-guides/some-setup-guide/index.md`

    ```md
    ---
    title: "Some setup guide"
    description: ""
    lead: ""
    date: 2023-02-10T14:41:39+01:00
    lastmod: 2023-02-10T14:41:39+01:00
    draft: false
    images: []
    type: docs
    menu:
      articles:
        parent: "setup-guides"
        identifier: "some-setup-guide"
    weight: 200
    toc: true
    ---
    ```

**Reply by h-enk** [reply]
In stead of copy-and-pasting you could also use the [create](https://getdoks.org/docs/overview/commands/#create) command (will also generate fresh dates), like so

```bash
npm run create articles/general/other-articles/index.md
```

```bash
npm run create articles/setup-guides/_index.md
```

```bash
npm run create articles/setup-guides/some-setup-guide/index.md
```

**Comment by ghost** [comment]
I am not sure how to reproduce the problem exactly I am experiencing.
After cloning this repository everything seems fine. So I worked further.
Went according to your steps:
```
---
title: "Some other article"
description: ""
lead: ""
date: 2023-02-10T14:41:39+01:00
lastmod: 2023-02-10T14:41:39+01:00
draft: false
images: []
type: docs
menu:
  articles:
    parent: "general"
    identifier: "some-other-article"
weight: 220
toc: true
---

TODO
```

But this is what awaits me on website.
![image](https://user-images.githubusercontent.com/49745287/218251354-bc95b59a-391f-4c6f-9193-605219efaa24.png)

I tried running  `npm run clean` and then it worked.
Is it a cache problem? Should I be always running the clean command?

**Comment by h-enk** [comment]
> Is it a cache problem? Should I be always running the clean command?

Yes it is. Not always, but to be sure that you're seeing your changes, yes, you should
